src/sentry/lang/java/processing.py

```diff
@@ -81,14 +82,16 @@ def _handles_frame(frame: dict[str, Any], platform: str) -> bool:
     "Returns whether the frame should be symbolicated by JVM symbolication."
 
     return (
-        "function" in frame and "module" in frame and (frame.get("platform") or platform) == "java"
+        "function" in frame
+        and "module" in frame
+        and (frame.get("platform", None) or platform) == "java"
```

Comment:
This can be `frame.get("platform", platform)`

Reply:
I suspect something sets `"platform": null`